cirq-core/cirq/work/sampler_test.py

```diff
@@ -324,20 +324,23 @@ def test_sampler_sample_expectation_values_multi_param():
     assert np.allclose(results, [[1], [-1], [1]])
 
 
-def test_sampler_sample_expectation_values_complex_param():
+def test_sampler_sample_expectation_values_complex_param() -> None:
     a = cirq.LineQubit(0)
     t = sympy.Symbol('t')
     sampler = cirq.Simulator(seed=1)
     circuit = cirq.Circuit(cirq.global_phase_operation(t))
     obs = cirq.Z(a)
     results = sampler.sample_expectation_values(
-        circuit, [obs], num_samples=5, params=cirq.Points('t', [1, 1j, (1 + 1j) / np.sqrt(2)])
```

Comment:
`cirq.Points` expects a [sequence of non-complex values](https://github.com/dstrain115/Cirq-1/blob/b4919588b0bd15a21efb351ada70dfcae8a5c2ee/cirq-core/cirq/study/sweeps.py#L485-L490).  
Should we adjust that annotation to allow complex values?

Reply:
I don't think so.  That would have a lot of downstream effects, since the methods that consume them expect floats.  (For instance, resolution of a gate exponent).

I think we should just treat this as a non-standard and non-supported use of the sweep.